src/components/gamesList.js

```diff
@@ -168,78 +168,81 @@ export default function GamesList({ token, games, listType, listTitle }) {
     console.log(games);
     return (
         <div>
-            <div>{listTitle}</div>
-            {games.map((game) => (
-                <div className="game" key={game.id}>
-                    <button
-                        onClick={() => {
-                            handleOpenModal(game);
-                        }}
-                    >
-                        more details
-                    </button>
-                    <div>Location: {game.location_info.park_name}</div>
-                    <div>
-                        {DateTime.fromISO(game.date).toLocaleString({
-                            weekday: "short",
-                            month: "short",
-                            day: "numeric",
-                        })}{" "}
-                        at{" "}
-                        {DateTime.fromISO(game.time).toLocaleString(
-                            DateTime.TIME_SIMPLE
-                        )}
+
+            
+    <div>{listTitle}</div>
+    {games.map((game) => (
+        <div className="game-card" key={game.id}>
+        <div>Location: {game.location_info.park_name}</div>
+            <div>
+                {DateTime.fromISO(game.date).toLocaleString({
+                    weekday: "short",
+                    month: "short",
+                    day: "numeric",
+                })}{" "}
+                at {" "}
+                {DateTime.fromISO(game.time).toLocaleString(
+                    DateTime.TIME_SIMPLE
+                    )}
+
                     </div>
+                    
+                    
+            <button-showmore
+                onClick={() => {
+                    handleOpenModal(game);
+                }}
+            >Show more</button-showmore>
```

Comment:
I haven't seen specific types of buttons like button-showmore and button-join before. Is this like having <button className="button-showmore">  ?

Reply:
yeah, in order to give them style I renamed those buttons which are actually working as before .